app/assets/stylesheets/finder_frontend.scss

```diff
@@ -308,4 +306,82 @@
       }
     }
   }
+
+  .facet-tags {
+    margin-top: $gutter-one-third;
+    margin-bottom: $gutter;
+  }
+
+  .facet-tags__group {
+    padding: 5px;
+
+    &:nth-child(odd) {
+      background-color: $grey-4;
+    }
+  }
+
+  .facet-tags__wrapper {
+    vertical-align: middle;
+    margin: 5px 0;
+
+    @include media(tablet) {
+      display: inline-block;
+    }
+  }
+
+  .facet-tags__wrapper:first-of-type .facet-tags__preposition:first-of-type {
+    font-weight: bold;
+
+    &::first-letter {
```

Comment:
Is this rule necessary? Looking at the source of the example heroku page it seems to be coming through with an uppercase first letter already.

I have a similar question about line 343 `text-transform: lowercase;` but I guess this is related?

Reply:
Yes, the news and communications finder only has facet labels that are a single word, so it isn't needed. But other finders, like [aaib-reports](https://finder-frontend-pr-739.herokuapp.com/aaib-reports?keywords=&date_of_occurrence%5Bfrom%5D=2018&date_of_occurrence%5Bto%5D=), have the label for date as "Occurred After" (for example).